src/main/java/com/dataguise/saas/DataOrientedSaasUiApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.dataguise.saas;
+
+import org.apache.log4j.Logger;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class DataOrientedSaasUiApplication {
+    Logger logger = Logger.getLogger(DataOrientedSaasUiApplication.class);
+	public static void main(String[] args) {
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.FinalParametersCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/DataOrientedSaasUiApplication.java L10
    
    Parameter args should be final.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a2002267238>*